Quest Behaviors/ForcedMount.cs

```diff
@@ -99,23 +99,11 @@ public ForcedMount(Dictionary<string, string> args)
             if (!Flightor.CanFly)
                 return false;
 
-            if (StyxWoW.Me.Class == WoWClass.Druid && (SpellManager.HasSpell("Flight Form") || StyxWoW.Me.Level >= 58))
-            {
-                if (SpellManager.CanCast("Flight Form"))
-                { SpellManager.Cast("Flight Form"); }
-
-                else if (SpellManager.CanCast("Travel Form"))
-                { SpellManager.Cast("Travel Form"); }
-            }
-
-            else
-            {
-                Mount.FlyingMounts.First().CreatureSpell.Cast();
-                await Coroutine.Wait(3000, () => StyxWoW.Me.Mounted);
-            }
+            Flightor.MountHelper.MountUp();
+            await Coroutine.Wait(3000, () => StyxWoW.Me.Mounted);

```

Comment:
MountUp already waits for the mount, so just get rid of this wait. We will clean all of this up a bit
in the near future.